package.json

```diff
@@ -40,7 +40,7 @@
     "@makerdao/multicall": "^0.12.0",
     "@metamask/detect-provider": "^1.2.0",
     "@stackoverflow/stacks": "^0.74.0",
-    "@theconvospace/sdk": "^0.3.29",
+    "@theconvospace/sdk": "^0.3.37",
```

Comment:
Updating a minor version shouldn't have such a large amount of diff (52k) in `package-lock.json` :thinking: 

I tried it locally with much less changes
![image](https://user-images.githubusercontent.com/30693990/161046432-2a2bb929-230a-4322-9674-e4250716d9ed.png)

Reply:
Reverted that change for now, as it not really nesessary for this PR.